src/core/export/csv.js

```diff
@@ -15,8 +15,10 @@ export class Csv {
 		const values = [];
 		let head = [];
 		for (let column of columns) {
-			values.push(valueFactory(column));
-			head.push(escape(column.title));
+			if (column.key !== 'rowNumber') {
```

Comment:
```class``` property is really good. But maybe it should be defined as 'data' in DataColumnModel? I'm confused a bit that it's declared in base class